travis/tensorflow_integration_tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -x
+
+# https://stackoverflow.com/questions/42218009/how-to-tell-if-any-command-in-bash-script-failed-non-zero-exit-status/42219754#42219754
+# Set err to 1 if pytest failed.
+error=0
+trap 'error=1' ERR
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+cd "$GIT_ROOT" || exit
+
+pip install tensorflow
```

Comment:
should we fix the TensorFlow version here?

Reply:
This is an interesting question. Just for comparison, the versions were indeed fixed for [`pytorch_integration_tests.sh`](https://github.com/bentoml/BentoML/blob/master/travis/pytorch_integration_tests.sh), but not for [`onnx_integration_tests.sh`](https://github.com/bentoml/BentoML/blob/master/travis/onnx_integration_tests.sh).

Is the worry that future API changes to TensorFlow might break the tests?

Reply:
@joshuacwnewton yes I think that is the concern, usually, it is best practice to pin the version so that the tests do not break due to external changes.   Although in current stage of the bentoml project, it is nice for us to get notified when a newer version of TensorFlow breaks our integration. Ideally, we should run a compatibility matrix to determine which are the versions of Tensorflow APIs that BentoML can support.

But before we get there, would be great if you could pin the version here to the current TensorFlow stable release. I think the onnx test will need to do the same as well. cc @yubozhao

Reply:
Thank you for explaining things so clearly. The reasoning makes a lot of sense! :)